packages/react-storage/jest.config.js

```diff
@@ -2,7 +2,10 @@ module.exports = {
   collectCoverage: true,
   collectCoverageFrom: ['<rootDir>/src/**/*.(ts|tsx)'],
   // ignore coverage for top level "export", version and style files
-  coveragePathIgnorePatterns: ['<rootDir>/src/(index|styles|version).(ts|tsx)'],
+  coveragePathIgnorePatterns: [
+    '<rootDir>/src/(index|styles|version).(ts|tsx)',
+    '<rootDir>/src/.*?/index.(ts|tsx)',
```

Comment:
It wasn't ignoring nested index.ts files, lowering the coverage by 2% :)

Reply:
Would prefer we do not do this in case actual code ends up in an `index` file

Reply:
That's true, we do have components in some index.tsx files in the ui-react package.